### ISSUE TITLE
Add sunrise, sunset, dusk and sawn

### ISSUE THREAD

**Issue by cloneofghosts** [Pirate-Weather/pirateweather#154]
### Discussed in https://github.com/Pirate-Weather/pirateweather/discussions/144

<div type='discussions-op-text'>

<sup>Originally posted by **markandersonus** January 25, 2024</sup>
Would it be possible to add these: sunrise, sunset, dusk and sawn</div>

**Comment by alexander0042** [comment]
Yes! Let's add this one to the board. I use [Astral](https://astral.readthedocs.io/en/latest/) for sunrise and sunset, so since it's already calculating that stuff, it's easy to add in.

**Comment by alexander0042** [comment]
This is live! Added dusk (`duskTime`) and dawn (`dawnTime`) in to v2.0i; however, since they're not in the Dark Sky fields, you'll have to add "version=2" to the URL in order to see them.

Thanks again for this suggestion!

**Comment by cloneofghosts** [comment]
@markandersonus this has been added to the latest version of the 2.0 development version of the API. Since this has been added I'm going to close this issue.
